build/ci/Makefile

```diff
@@ -17,3 +17,17 @@ ci/build:
 
 ci/push:
 	docker push $(ci_image)
+
+bench/prepare:
+	pip install -r ./build/ci/requirements.txt
+
+old_bench ?= benchmark-master.txt
+new_bench ?= benchmark-pr.txt
+alert_threshold ?= 5
+aggregation_method ?= "mean"
+
+bench/compare:
+	@python ./build/ci/compare_benchmarks.py \
+	  $(old_bench) $(new_bench) \
+	  --alert-threshold $(alert_threshold) \
+	  --aggregation $(aggregation_method)
```

Comment:
This doesn't look like a CI build script (but folder name is build/ci). If you want to keep the make commands, you need to move them to makefile with bench test command and make them work locally. It is probably easier and more logical to make these commands directly in the workflow.